src/parser/druidsql.pegjs

```diff
@@ -568,25 +568,37 @@ SqlInParens = OpenParen leftSpacing:_? ex:Sql rightSpacing:_? CloseParen
   return ex.addParens(leftSpacing, rightSpacing);
 }
 
-SqlLiteral = lit:(Number / SingleQuotedString)
+SqlLiteral = lit:(Number / SingleQuotedString )
 {
   return new sql.SqlLiteral(lit);
 }
+/ BooleanLiteral
 / SqlInParens
 
+NullLiteral = value: "NULL"i {
+    return new sql.SqlLiteral({value: value, stringValue: value, quotes:''});
+}
+
+BooleanLiteral = value: ("TRUE"i/"FALSE"i) {
+  return new sql.SqlLiteral({value: value, stringValue: value, quotes:''});
+}
+/NullLiteral
```

Comment:
Why is `BooleanLiteral` defined as `.../NullLiteral` just use `NullLiteral` in `SqlLiteral`